contracts/Vault.sol

```diff
@@ -45,6 +45,8 @@ contract Vault is
     // Constants
     //
 
+    bytes32 public constant MANAGER_ROLE =
+        0x241ecf16d79d0f8dbfb92cbc07fe17840425976cf0667f022fe9877caa831b08; // keccak256("MANAGER_ROLE");
```

Comment:
can we use `keccak256("MANAGER_ROLE")` instead of the hardcoded hash?
it's evaluated at compile-time, so doesn't cost extra gas, and prevents mistakes. it's hard to validate the hash is correct like this

Reply:
fixed